wiki/beginners.md

```diff
@@ -0,0 +1,450 @@
+
+You can instantiate a
+[BaseClient](https://remotestorage.github.io/remotestorage.js/files/baseclient-js.html)
+for a specific path using the
+[remoteStorage.scope()](https://remotestorage.github.io/remotestorage.js/files/remotestorage-js.html#RemoteStorage.scope)
+function, and then use BaseClient methods directly in your code. This is
+meant primarily for development, but might make sense to use in shipped
+code in small apps as well.
+
+``` javascript
+remoteStorage.access.claim('bookmarks', 'rw');
+var baseClient = remoteStorage.scope('/bookmarks/archive/');
+
+baseClient.getListing('/').then(function(listing){
+  console.log(listing); // logs the directory listing of '/bookmarks/archive/'
+});
+```
+
+### Configuring caching
+
+#### Caching strategies
+
+For each subtree, you can set the caching strategy to ‘ALL’, ‘SEEN’
+(default), and ‘FLUSH’.
+
+  - ALL  
+    Once all outgoing changes have been pushed, sync will start
+    retrieving nodes to cache pro-actively. If a local copy exists of
+    everything, it will check on each sync whether the ETag of the root
+    folder changed, and retrieve remote changes if they exist.
+  - SEEN  
+    Same as ALL but only for documents and folders that have been either
+    read from or written to at least once since connecting to the
+    current remote backend, plus their parent/ancestor folders up to the
+    root (to make tree-based sync possible).
+  - FLUSH  
+    Only caches outgoing changes, and forgets them as soon as they have
+    been saved to remote successfully.
+
+#### Example
+
+[remoteStorage.Caching](https://remotestorage.github.io/remotestorage.js/files/caching-js.html)
+manages caching configuration. Enabling caching for a given path with
+strategy 'ALL' is as simple as:
+
+``` javascript
+remoteStorage.caching.enable('/bookmarks/archive/');
+```
+
+### Adding the connect widget
+
+![Connect widget](widget.png "Connect widget")
+
+The remoteStorage widget is the easiest way to allow your users to
+connect to their personal storage server. By default it is part of every
+app that uses remoteStorage.js. So every time a user sees the widget,
+they know that they can use their own storage with an app. You can
+customize the widget's appearance to better match your app's design, of
+course. It is as simples as:
+
+``` javascript
+remoteStorage.displayWidget();
+```
+
+*Hint: The widget should be called only AFTER claiming access for a
+category.*
+
+## Cordova
+
+Since version 0.12.0, you can use remoteStorage.js in packaged
+[Cordova](https://cordova.apache.org/) apps on a variety of platforms.
+Support is built into the library by default, but there are a couple of
+extra steps to make it work:
+
+### Install the Cordova [InAppBrowser plugin](https://github.com/apache/cordova-plugin-inappbrowser)
+
+The OAuth dialog for the user will be opened in an in-app-browser window
+in your app. Install the plugin like so:
+
+`cordova plugin add cordova-plugin-inappbrowser`
```

Comment:
Suggestion to use a fenced code block here, either with four spaces before the line, or three tick marks before and after.